app/controllers/bookmarked_posts_controller.rb

```diff
@@ -0,0 +1,19 @@
+class BookmarkedPostsController < ApplicationController
+  def index
+    @posts = Post.joins(:bookmarked_posts).where(bookmarked_posts: { user_id: current_user.id })
+  end
+
+  def bookmark
+    bookmarked = false
+    @bookmark = BookmarkedPost.find_by(user: current_user, post_id: params[:bookmarked_post_id])
```

Comment:
Nepamaniju vel vienu lietu. Skaties tev @bookmark ir instances mainigais, bet talak nekur tu to neizmanto, tāpēc vari taja vietā izmanto parasto